app/views/posts/index.html.erb

```diff
@@ -1 +1,47 @@
-<h2> Post interactions for a given user </h2>
\ No newline at end of file
+<h2 class='heading'> Post interactions for a given user </h2>
+<div class='all-posts-body'>
+    <div class='u-detail-top'>
+        <div class='u-d-img'>
+            <%= image_tag 'home-view.png', width: '150px', height: '150px' %>
+        </div>
+        
+        <div class='u-detail-user'>
+            <h1 class='u-detail-user-name'>
+                <%= @user.name  %>
+            </h1>
+            <p class='u-detail-user-counter'>
+                Number of posts: <%= @user.posts_counter %>
+            </p>
+        </div>
+    </div>
+
+    <div class='u-detail-rec-posts'>
+        <% @posts.each  do |post| %>
+            <div class='u-detail-rec-posts-card'>
+                <h2>
+                    <%= link_to "#{post.title}", user_post_path(@user.id, post.id), class: 'post-link' %>
+                </h2>
+                <p>
+                    <%= post.text %>
+                </p>
+
+                <div class='com-lik-counters'>
+                    <p>Comments: <%= post.comments_counter %></p>
+                    <p>Likes: <%= post.likes_counter %></p>
+                </div>
+            </div>
+
+            <div class='post-comments'>
+                <% post.show_recent_5_comments.each do |comment| %>
+                    <h3>
+                        <%= comment.user.name %>: <%= comment.text  %>
+                    </h3>
+                <% end %>
+            </div>
+        <% end %>
```

Comment:
- [ ] Kindly note that `<%= comment.user.name%>` this relationship does not exist in your comment table, if you want to get the user name kindly refer to the `<%= @user.name %>` which is defined in the post#show controller. I am getting this error :point_down: 

![image](https://user-images.githubusercontent.com/65068771/156240628-e1953fb0-fdd1-43dc-a555-8c81d556846e.png)